index.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width">
+    <link rel="stylesheet" href="style.css">
+    <title>Document</title>
+</head>
+<body>
+
+    <header>
+        <div>
+            <nav id="head">
+                <p>Safar Ali</p>
+                <img src="../Portfolio/images/Union.png" alt="burger-icon">
+            </nav>
+            <div id="name">
+                <h2>I'm Safar Ali <br> A thriving <br> Software Developer </h2>
+            </div>
```

Comment:
- [ ] The main title( I'm Safar Ali ... ) should be a  `h1` tag. Please,  kindly fix it.